mlflow/langchain/databricks_dependencies.py

```diff
@@ -162,7 +162,12 @@ def _detect_databricks_dependencies(lc_model, log_errors_as_warnings=True) -> Li
     try:
         dependency_list = []
         _traverse_runnable(lc_model, dependency_list, set())
-        return dependency_list
+        # Filter out duplicate dependencies so same dependencies are not added multiple times
+        unique_dependencies = []
+        for dependency in dependency_list:
+            if dependency not in unique_dependencies:
+                unique_dependencies.append(dependency)
+        return unique_dependencies
```

Comment:
Can we do"
```suggestion
        return return list(set(dependency_list))
```

Reply:
Unfortunately I had tried that, using a set, the keys are hashed. but the `DatabricksVectorSearchIndex` for example can't be hashed